rero_ils/modules/holdings/api.py

```diff
@@ -261,28 +262,24 @@ def _get_next_issue_display_text(cls, patterns):
         :param patterns: List of a valid holdings patterns.
         :returns: A display text of the next predicted issue.
         """
-        texts = []
-        for pattern in patterns:
-            text = []
+        issue_data = {}
+        for pattern in patterns.get('values', []):
             levels = pattern.get('levels', [])
+            pattern_name = pattern.get('name', '')
+            level_data = {}
             for level in levels:
```

Comment:
Can you use `for level in pattern.get('levels', []):` (and remove line 267) ?
it seems `levels` variable isn't use anywhere else.

Reply:
done